CMSgov/mint-app
# [EASI-2677] task list ready for review by user

# EASI-2677

## Changes and Description

### Back End

- add readyForClearanceByUserAccount to all task list sections

### Front End 

- Added `readyForClearanceByUserAccount.commonName` to `GetClearanceStatus.ts` query and typers
- Updated `SectionClearanceLabel` prop names to be more readable and to take in `commonName`



## How to test this change

1. Navigate to Prepare for Clearance
2. Mark the task list item to be ready for clearance
3. Make sure the Username is showing, not ID or completely missing



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
